### DISCUSSION TITLE
i update a csv file, but when run the harvest, it don't update

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3894] [Q&A]
Hello everyone, I made a change in the csv file, I am running the harvest, but it only shows processed, and when running the datastore_import it does not update the info, it remains with the old one, how can I solve this or what could it be? Thanks a lot

![image](https://user-images.githubusercontent.com/22879539/209733055-cd7b6359-185c-4fa6-a629-b5d732352d35.png)

**Comment by tridoxx** [comment]
okay i found this on documentatión https://demo.getdkan.org/modules/contrib/dkan/docs/guide_datastore_settings.html

**Comment by janette** [comment]
The downloadURL value (either a change to the file name or a completely different URL) would need to change to trigger an new import. Or you can enable the modified date to also be a trigger. Then any change to that value will also cause the datastore to be regenerated.

**Comment by guille-amarilla** [comment]
Hello @janette, how can I make the datastore to the regenerated automatically, like, everyday or something like that? Because I have .csv link that have data updated almost everyday, and I want to automatically update in the datastore too without having to edit the dataset every day. Thanks in advance.

**Reply by janette** [reply]
see #4108

**Comment by janette** [comment]
@tridoxx yes if your data resource files update without a change to the file name or file path you will need to set up a different field to trigger a fresh import of the data on the datastore settings screen (/admin/dkan/datastore). The **Last Update (modified)** is what we use on the sites we manage. When this value changes, the previous datastore table will be dropped and a new table generated from the new data.
